src/core/observables/ParticleTraits.hpp

```diff
@@ -0,0 +1,37 @@
+#ifndef OBSERVABLES_PARTICLE_TRAITS
```

Comment:
missing copyright header

Reply:
is this a PR review criterium? This is check before every release... actually we have tooling for that, so if we want to have it as a PR check, we could do that together with the other formatting checks.

Reply:
We should consider enforcing it in CI for the subset of C++/CUDA files. Tracking down the year range after the fact is tedious (the first year depends on whether the file was copied from another file, or restored from a reverted commit) and unnecessarily slows down the release workflow.